packages/algob/test/smart-contracts/escrow-account.ts

```diff
@@ -0,0 +1,78 @@
+import { Interpreter } from "algorand-js/src";
+import { ERRORS } from "algorand-js/src/errors/errors-list";
+import { expectTealErrorAsync } from "algorand-js/test/helpers/errors";
+import { assert } from "chai";
+
+import { useEnvironment } from "../helpers/environment";
+import { useFixtureProject } from "../helpers/project";
+import { Runtime } from "../helpers/runtime";
+import { MockAccount } from "../mocks/state";
+
+describe("Algorand Smart Contracts", function () {
```

Comment:
I think this test should be an integration test in `algorand-js` (eg: algorand-js/tests/integration/escrow-account.ts). It doesn't use `algob`.

In `algob`, in the future we will add a functional test - a fixture algob project with teal code in `/assets`, tests in `/tests`.

Reply:
Alright.